atomicapp/cli/main.py

```diff
@@ -381,6 +395,28 @@ def create_parser(self):
             help='The name of a container image containing an Atomic App.')
         gena_subparser.set_defaults(func=cli_genanswers)
 
+        # === "index" SUBPARSER ===
+        index_subparser = toplevel_subparsers.add_parser(
+            "index", parents=[globals_parser])
+        index_action = index_subparser.add_subparsers(dest="index_action")
+
+        index_list = index_action.add_parser("list")
+        index_list.add_argument(
+            "--update",
```

Comment:
so what do you think about having this as a separate subcommand rather than an option to `--list`? seems like `atomicapp index update` would be more natural than `atomicapp index list --update`.